techmatters/hrm
# CHI-1912: E2E tweaks

Bag of minor tweaks to get the resources backend up and running E2E in Aselo Development

## Description

* Moved resources-search-config into a packages directory within resources-domain
* Updated generic Dockerfiles to use wildcards on domain specific packages directories. 
* Prevented resources lambdas deploying to regions where resources isn't enabled by explicitly blocking them out in the environment-region-map.json
* Reimplemented the 'filters as search terms' mechanism to use a `simple_query_string` rather than a `multi_match` - can control the AND / OR logic and it seems to give more predictable results


### Checklist

### Related Issues

CHI-1912, CHI-1735

### Verification steps

Deploy all the lambdas & HRM to development
Delete the records in the resources."Accounts" table
'Test' the resources=import-producer-as lambda (input doesn't matter)
Review DataDog logs, it should show the records being imported to the DB and submitted for indexing
Now, with the https://github.com/techmatters/flex-plugins/pull/1461 version of flex deployed, test some filtered searches - they should give back something reasonable, if not perfect
